src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -100,14 +104,22 @@ def __init__(self, url, max_schemas_per_subject=1000, ca_location=None, cert_loc
         # subj => { schema => version }
         self.subject_to_schema_versions = defaultdict(dict)
 
+        ca_certs = conf.pop('ssl.ca.location', None)
```

Comment:
Can this be empty? What is the behaviour in that case? Ideally it should refer to the default paths.

Reply:
This cannot be empty if you want a SSL connection, not even in the previous requests module.